### PR TITLE
Hts 1240 not eligible page content changes

### PR DESCRIPTION
**Pull request by nubz**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/views/register/not_eligible.scala.html [hunk 1]
```diff
@@ -37,11 +37,11 @@ <h1 class="heading-large" id="hts-not-eligible-title">@Messages("hts.core.not-el
             <p>@messages("hts.core.not-eligible.uc.p-1")</p>
 
             <p>@messages("hts.core.not-eligible.uc.p-2")
-            <a href=@routes.IntroductionController.getEligibility().url
+            <a href=@{appConfig.govUkNotEligibleURL}
             class="ga-track-event"
             data-ga-event-category="links"
             data-ga-event-action="Help to Save"
-            data-ga-event-label="Not eligible page">@Messages("hts.core.not-eligible.uc.link-1")</a>@Messages("hts.core.not-eligible.uc.p-2-2")</p>
+            data-ga-event-label="Not eligible page">@Messages("hts.core.not-eligible.uc.link-1")</a>@Messages("hts.core.not-eligible.uc.p-2-2").</p>
```
Comment (Jackie1988) [inline app/uk/gov/hmrc/helptosavefrontend/views/register/not_eligible.scala.html]: Can the full stop be put inside the messages file? (very picky I know :) )

File: conf/application.conf [hunk 2]
```diff
@@ -202,4 +202,6 @@ nino-logging.enabled = true
 # if enabled the account creation cap check is done before the DES Eligibility Check API call
 enable-early-cap-check = true
 
-gov-uk.url="https://www.gov.uk"
\ No newline at end of file
+gov-uk.url="https://www.gov.uk"
+
+gov-uk.not-eligible.url="https://www.tax.service.gov.uk/help-to-save/not-eligible"
```
Comment (Jackie1988) [inline conf/application.conf]: Not sure if this is needed, can we not just direct the user directly to our page?

File: app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala [hunk 3]
```diff
@@ -95,4 +95,5 @@ class FrontendAppConfig @Inject() (override val runModeConfiguration: Configurat
   val reportAProblemNonJSUrl: String = s"$contactBaseUrl/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
   val betaFeedbackUrlNoAuth: String = s"$contactBaseUrl/contact/beta-feedback-unauthenticated?service=$contactFormServiceIdentifier"
   val govUkExitURL: String = getString("gov-uk.url")
+  val govUkNotEligibleURL: String = getString("gov-uk.not-eligible.url")
```
Comment (Jackie1988) [inline app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala]: We could add something like ```val notEligibleUrl: String = s"${getUrlFor("help-to-save-frontend")}/not-eligible" ```

Comment (nubz) [reply app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala]: removed the hard coded full stop.

Comment (nubz) [reply app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala]: link text updated to correct value as per story www.gov.uk/get-help-savings-low-income/eligibility

**Comment by MarcusQA** [comment]
Agreed with Alistair to close this PR to coordinate work with availability of GOV.UK pages.

**Pull request closed without merging**